go/service/gregor.go

```diff
@@ -98,6 +224,10 @@ func (g *gregorHandler) ShouldRetryOnConnect(err error) bool {
 func (g *gregorHandler) BroadcastMessage(ctx context.Context, m gregor1.Message) error {
 	g.G().Log.Debug("gregor handler: broadcast: %+v", m)
 
+	// Send message to local state machine
+	g.gregorCli.ConsumeMessage(m)
```

Comment:
Expose state machine on g itself instead of this other way which is too confusing.